source/Calamari.Common/Calamari.Common.csproj

```diff
@@ -65,7 +65,7 @@
         <PackageReference Include="Octostache" Version="3.3.0" />
         <PackageReference Include="SharpCompress" Version="0.24.0" />
         <PackageReference Include="XPath2" Version="1.0.12" />
-        <PackageReference Include="YamlDotNet" Version="8.1.2" />
+        <PackageReference Include="YamlDotNet" Version="13.1.0" />
```

Comment:
Bumping this to the latest version. 
To be more specific, we want to include this fix: https://github.com/aaubry/YamlDotNet/commit/c8a70f0eebf2889238dc35047b70705a1bf61ce3

Otherwise the output block comment will lose its indentation.